### DISCUSSION TITLE
Do you have a plan to add automatic upgrading?

### DISCUSSION THREAD

**Discussion by junyuan-qi** [UdaraJay/Pile#75] [General]
Key Feature:
- Automatic Detection: Leverages GitHub's APIs to monitor for new releases or commits, ensuring that the software checks for updates at configurable intervals.
- Seamless Upgrade: Implements a smooth and user-friendly upgrade process, allowing for silent background updates or user-prompted installations based on the application's nature and user preferences.

**Comment by UdaraJay** [comment]
this is built into Pile from v0.9.3 onwards :)
